batcher/aligned/.env.bk

```diff
@@ -0,0 +1 @@
+PRIVATE_KEY=0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d
```

Comment:
is it ok to commit this?

Reply:
Its an anvil private key, removed it anyways cause its not relevant for PR